aws/resource_aws_chime_voice_connector_group.go

```diff
@@ -0,0 +1,177 @@
+package aws
+
+import (
+	"context"
+	"log"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/chime"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
+)
+
+func resourceAwsChimeVoiceConnectorGroup() *schema.Resource {
+	return &schema.Resource{
+		CreateContext: resourceAwsChimeVoiceConnectorGroupCreate,
+		ReadContext:   resourceAwsChimeVoiceConnectorGroupRead,
+		UpdateContext: resourceAwsChimeVoiceConnectorGroupUpdate,
+		DeleteContext: resourceAwsChimeVoiceConnectorGroupDelete,
+
+		Importer: &schema.ResourceImporter{
+			StateContext: schema.ImportStatePassthroughContext,
+		},
+
+		Schema: map[string]*schema.Schema{
+			"connector": {
+				Type:     schema.TypeList,
```

Comment:
Does the order of the connectors matter, and will they consistently be returned in the same order? If not, you should use `TypeSet` here instead.